bgpd/bgp_fsm.c

```diff
@@ -1068,6 +1114,10 @@ int bgp_stop(struct peer *peer)
 	safi_t safi;
 	char orf_name[BUFSIZ];
 	int ret = 0;
+	struct bgp *bgp = peer->bgp;
+	struct graceful_restart_info *gr_info = NULL;
```

Comment:
by the way, I would use `gri` as a name everywhere to keep format like `bgp_damp_config` => `bdc`, `bgp_path_info` ==> `bpi`, etc.

Reply:
Not sure, if this is a coding conversion or not. However i tried to change gr_info to gri as you suggested, then i saw there are around 80 places to change across around 10 different files. This would involve changing the well tested code too much. So i am leaving it as it is.